### PR TITLE
#8 The smallest positive number that is evenly divisible by all of the numbers from 1 to 20

### PR DESCRIPTION
**Pull request by sanishmaharjan**
Please code review.

### CODE REVIEW

File: src/com/lftechnology/java/training/sanish/multiplenumber/Main.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package com.lftechnology.java.training.sanish.multiplenumber;
+
+import java.util.logging.Logger;
+
+/**
+ * Class Main
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ *
+ */
+public class Main {
```
Comment (utsabneupane) [inline src/com/lftechnology/java/training/sanish/multiplenumber/Main.java]: use relevant class Name here

File: src/com/lftechnology/java/training/sanish/multiplenumber/MultiplierNumber.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ */
+public class MultiplierNumber {
+	/**
+	 * Return smaller multiple number form given range
+	 * @param int form first number that have to find multiple
+	 * @param int to last number that have to find multiple
+	 * @return int smaller multiple number
+	 * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+	 */
+	public static int getSmallerMultipleNum(int form, int to) {
```
Comment (nimesh-mishra) [inline src/com/lftechnology/java/training/sanish/multiplenumber/MultiplierNumber.java]: first argument name should be "from". :ghost:

File: src/com/lftechnology/java/training/sanish/multiplenumber/Main.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	public static void main(String args[]) {
+		final Logger LOGGER = Logger.getLogger(Main.class.getName());
+		try {
+			LOGGER.info("The smallest positive number that is divisible by all of the numbers from 1 to 20 : "
```
Comment (Satish-Gupta) [inline src/com/lftechnology/java/training/sanish/multiplenumber/Main.java]: use parameterized logger

File: src/com/lftechnology/java/training/sanish/multiplenumber/Main.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+					+ MultiplierNumber.getSmallerMultipleNum(1, 20));
```
Comment (Satish-Gupta) [inline src/com/lftechnology/java/training/sanish/multiplenumber/Main.java]: defining 1 and 20 as constants and then passing those values in the method is recommended

**Pull request merged**